docs/docs/guides/kubescape-gha/index.md

```diff
@@ -63,7 +63,7 @@ First, create a new branch called dev:
 
 `git checkout -b dev`
 
-We’re going to do a very simple manifest that will deploy an Nginx pod to start. Although simple, it still contains several security vulnerabilities, as you can see. Create a file called nginx.yml in the root directory of your repository and add the following:
+We’re going to do a very simple manifest that will deploy a Nginx pod to start. Although simple, it still contains several security vulnerabilities, as you can see. Create a file called nginx.yml in the root directory of your repository and add the following:
```

Comment:
I've seen NGINX or nginx (on their docs) but never mixed case. 
Please select according to your preference.

Reply:
done